README.md

```diff
@@ -27,6 +27,11 @@ Demonstrate calling the method with an argument of "young prince".
 Write your code here:
 ```ruby
 # code here
+def offerRose person
```

Comment:
This is correct, but I wanted to point out that many rubyists would include the parens in a method definition for clarity, and rubyists prefer snake_case over camelCase!

```ruby
def offer_rose(person)
end
```